src/bentoml/serve.py

```diff
@@ -319,7 +324,12 @@ def serve_http_production(
     from ._internal.configuration.containers import BentoMLContainer
 
     working_dir = os.path.realpath(os.path.expanduser(working_dir))
-    svc = load(bento_identifier, working_dir=working_dir, standalone_load=True)
+
+    standalone_load: bool = False if development_mode else True
```

Comment:
Why are we loading standalone if in development mode?

Reply:
The old development code load in standalone.

Reply:
Hm, @parano @aarnphm do you remember why this was?

Reply:
I believe this is a hack around the import module issue. I don't think standalone_load makes sense

Reply:
after some code reading, I think `standalone_load=True` should be only used by runner/api server, so I remove this part